cql3/statements/alter_keyspace_statement.cc

```diff
@@ -75,7 +75,7 @@ void cql3::statements::alter_keyspace_statement::validate(query_processor& qp, c
 }
 
 future<std::pair<::shared_ptr<cql_transport::event::schema_change>, std::vector<mutation>>>
-cql3::statements::alter_keyspace_statement::prepare_schema_mutations(query_processor& qp, api::timestamp_type ts) const {
+cql3::statements::alter_keyspace_statement::prepare_schema_mutations(query_processor& qp, api::timestamp_type ts, std::vector<sstring>& warnings) const {
```

Comment:
Is warnings an output parameter?

If so make it part of the output tuple.

Reply:
done